contracts/interfaces/ISwap.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+interface ISwap {
+    struct Offer {
+        // 32 byte slot 1
+        address seller;
+        bool isOpen;
```

Comment:
When the `availableSize == 0`, will it `isOpen = false`? If that’s the case, should we just remove `isOpen` completely since it `availableSize` makes it redundant.

Reply:
Main use for `isOpen` is in case our vaults cant sell out the entire volume, so we can close the offering even when `availableSize != 0`